src/QsCompiler/SyntaxProcessor/ContextVerification.fs

```diff
@@ -257,15 +259,15 @@ let private followedByApply context =
     | _ -> false, [| (ErrorCode.MissingContinuationApply |> Error, context.Range) |]
 
 
-type ContextVerification = delegate of SyntaxTokenContext -> (bool * QsCompilerDiagnostic [])
+type ContextVerification = delegate of SyntaxTokenContext * bool -> (bool * QsCompilerDiagnostic [])
```

Comment:
I think there is some boolean blindness, maybe a new type would be clearer. Something like:

```fsharp
type DocumentKind = File | Notebook
```